template.js

```diff
@@ -467,5 +481,6 @@ if (data.tag_type == 'data_layer_event') {
 
 log(LOG_PREFIX + ' Tag is fired with Event name from data layer `'+ dataLayerEventName +'` and tag_type `'+ data.tag_type +'`');
 
-// inject the Journify JavaScript SDK
+
+// inject the Journify JavaScript SDK only on Init
 injectScript(JS_URL, onsuccess, onfailure, 'journify');
```

Comment:
you need to merge with main, the script is injected now only on init, not always